py_v_sdk/contract/atomic_swap_ctrt.py

```diff
@@ -162,3 +184,227 @@ async def get_tok_bal(self, addr: str) -> int:
             int: The balance of the token.
         """
         return await self._query_db_key(self.DBKey.for_token_balance(addr))
+
+    def maker_lock(
```

Comment:
Missing `async` here.

Please check & ensure `async` & `await` show up where it is requried.